app/models/post.rb

```diff
@@ -1,3 +1,16 @@
 class Post < ApplicationRecord
-  belongs_to :users
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+  after_save :update_post_counter
+
+  def recent_comments
+    Comment.where(post_id: id).order(created_at: :desc).limit(5)
+  end
+
+  private
+
+  def update_post_counter
+    user.increment!(:postsCounter)
+  end
```

Comment:
- [ ] **[OPTIONAL]** According to the ERD diagram, the correct naming for the posts counter is `posts_counter` and not `postsCounter`. Let's make use of the snake case ruby naming convention and also fulfill the project requirement. You can do this for the likes and comments counter too.